docs/guides/colors.md

```diff
@@ -0,0 +1,117 @@
+# Colors
+
+<!-- prettier-ignore-start -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [Contributing](#contributing)
+  - [Colors](#colors)
+    - [Gotchas](#gotchas)
+  - [Tokens](#tokens)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- prettier-ignore-end -->
+
+## Contributing
+
+When looking to make contributions to colors, you can find all the source files
+available to edit [in this directory](/packages/colors/src). There are a couple
+of files, namely:
+
+- `colors.js` which holds all the color information for the IBM Design Language
+- `tokens.js` which holds all the token information for the Carbon Design System
+
+You can make edits to either file in order to update a name or value of a color.
+
+### Colors
+
+When working with color names and values, you will be working in the
+[`colors.js`](/packages/colors/src/colors.js) file. This file is written in
+JavaScript and exports all the colors that developers could use from the IBM
+Design Language. A color is exported by writing the following:
+
+```js
+export const red10 = '#fff0f1';
+```
+
+`red10` represents the name of the color, in this case the color in the `red`
+swatch with a grade of `10` and `'#fff0f1'` represents its value. More often
+than not, the value is a HEX code value or an rgb/rgba value.
+
+You can update the value of any of the colors in this file by changing the HEX
+code. For example, we could update `red10` above to be:
+
+```js
+export const red10 = '#fff0f2';
+```
+
+You can update these values to any valid HEX code or rgb value. The important
+thing to remember is that the value is contained within the single quotes, `'`,
+otherwise the JavaScript program will not run.
+
+#### Gotchas
+
+- Make sure that every value is lowercase in the HEX code
+  - In other words, prefer `#ffffff` to `#FFFFFF`
+- Make sure that the value is contained within the single quotes, `'`
+  - For example, `export const red10 = '#fff0f1;` would not compile without the
+    ending `'` quote.
+
+### Tokens
```

Comment:
I'm wondering if it would be value to put a note somewhere in this section about why black, white, yellow, and orange are treated differently?  This may help get ahead of any questions/confusion as to why they are different.

Reply:
Good call! Will add a note soon when I run through this with some designers 😄